wpilibNewCommands/src/main/native/cpp/frc2/command/Command.cpp

```diff
@@ -205,6 +207,22 @@ void Command::InitSendable(wpi::SendableBuilder& builder) {
       "runsWhenDisabled", [this] { return RunsWhenDisabled(); }, nullptr);
 }
 
+CommandPtr operator<<(CommandPtr command, auto toRun) {
+  return command.AndThen(toRun);
+}
+
+CommandPtr operator<<(Command* command, auto toRun) {
+  return command->AndThen(toRun);
+}
+
+CommandPtr operator|(CommandPtr command1, CommandPtr command2) {
+  return frc2::cmd::Parallel(std::move(command1), std::move(command2));
+}
+
+CommandPtr operator&(CommandPtr command1, CommandPtr command2) {
```

Comment:
I don't really see how operator| and operator& map to parallel and race respectively.

Reply:
These are just what I am used to from Behaviours (my inspiration for said changes) I am open to changes to the used operators and also open to additions eg for deadline

Reply:
I'll comment these out for now